ssg/build_stig.py

```diff
@@ -7,14 +7,7 @@
 from .constants import XCCDF11_NS, stig_ns, stig_refs
```

Comment:
```suggestion
from .constants import XCCDF11_NS
```

Those other symbols are not used.